### PR TITLE
Updated notes on Dense layer using `linear` instead `sigmoid` and tf 2.1.0 compatible args

### PR DESCRIPTION
**Pull request by shern2**
1) Updated the notes on Dense layer using `sigmoid` activation, when it is actually using `linear` + `BinaryCrossentropy`'s  "from_logits" option.

2) The notebook installs TF 2.2.0 dev. However, for TF 2.1.0, "padded_shapes" is a mandatory agurment for the `padded_batch()` function. Of course, this change is subjective as it may affect readability of the tutorial.

### CODE REVIEW

**Comment by lamberta** [comment]
Thanks @shern2 
Please double-check that you've signed the CLA with the email used to author the commits: https://cla.developers.google.com/ 
Thanks

**Comment by shern2** [comment]
@googlebot I signed it!

**Comment by shern2** [comment]
Thanks @lamberta. I signed it, but didn't reply the googlebot. Just did.
